Kernel_23/doc/Kernel_23/CGAL/Bbox_2.h

```diff
@@ -18,6 +18,14 @@ class Bbox_2 {
 /// @{
 
 /*!
+introduces an \em empty bounding box with lower left
+corner point at \f$ (\infty, \infty) \f$
+and with upper right corner point at
+\f$ (-\infty, -\infty) \f$, \f$ \infty \f$ being
+`std::numeric<limits<double>::%infinity()`.
+*/
```

Comment:
There is a typo here: `std::numeric<limits<double>::%infinity()`, in `numeric_limits`.